src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let persons;
+
+  century === undefined
+    ? persons = people.filter((n) => n.sex === 'm')
+    : persons = people.filter((person) => person.sex === 'm'
+      && Math.ceil(person.died / 100) === century);
```

Comment:
use the ternary operator to decide which condition use to filter the array, don't repeat this part of the code
```persons = people.filter((person => )```